Print_A_Bot/djwifi/views.py

```diff
@@ -0,0 +1,109 @@
+import re
+from logging import getLogger
+from subprocess import call, check_output
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.shortcuts import redirect, render
+from django.utils.translation import ugettext as _
+from rest_framework.decorators import api_view
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from djwifi.forms import WifiForm
+from djwifi.iw_parse import call_iwlist, get_djwifi_list
+from general.utils_view import return_success_msg
+from controls.utils.system import call_sudo_command
+
+
+logger = getLogger(__name__)
+
+
+def wifi_home(request):
+    context = {
+        'configured': settings.WIFI_INTERFACE,
+    }
+    return render(request, 'djwifi/configure.html', context)
+
+
+@api_view(['GET', ])
```

Comment:
`@api_view(['GET', ])` started porting these to use Django Rest Framework

`api_view` is one of the easier ways to change an existing function that doesn't support a full REST structure

http://www.django-rest-framework.org/api-guide/views/